src/code/main.py

```diff
@@ -17,6 +17,7 @@
 import datetime
 import logging
 from typing import Iterable
+from pathlib import Path
```

Comment:
nit: Following the Google python style guide, import pathlib and use pathlib.Path.

(typing imports are an exception to that rule.)

Reply:
Got it.